tools/backup/backup.sh

```diff
@@ -172,15 +185,28 @@ function activate_aws() {
   fi
 }
 
+function activate_azure() {
+  echo "Activating azure credentials before beginning"
+  source "/credentials/credentials"
```

Comment:
considering how the `az` cli works this is a fine way to do it.   See earlier comment about documentation.   In the Azure section of the docs, we need to be explicit about the structure of the file (it's a bash script) and the required env vars.

Also, right under this line before az login, you should check the env to make sure that all of the things you need are defined, and auto-fail without doing az login if anything is missing.   This will give the user a clean error message if they've forgotten to include 1 of the env vars, rather than relying on az login, which will probably fail with something weird.

I just mean a simple checks like this: https://github.com/neo4j-contrib/neo4j-helm/blob/master/tools/backup/backup.sh#L8

Reply:
Added checks of variables.